tests/test_naming.py

```diff
@@ -38,6 +39,23 @@ def test_to_snake_case() -> None:
     assert to_snake_case("verlengingSluitingstijd1") == "verlenging_sluitingstijd_1"
     assert to_snake_case("numbers33inTheMiddle44") == "numbers_33_in_the_middle_44"
     assert to_snake_case("perJaarPerM2") == "per_jaar_per_m2"
+    assert to_snake_case("eersteHNId") == "eerste_h_n_id"
 
     with pytest.raises(ValueError):
         to_snake_case("")
+
+
+def test_snake_and_camel_case_functions_are_inverses() -> None:
+    """Prove that snake and camel case functions are inverses of eachother."""
+    # mind the lower case "i" after "33". It should be made upper case
+    assert to_snake_case(toCamelCase("per_jaar_per_m2")) == "per_jaar_per_m2"
+    assert to_snake_case(toCamelCase("test_name_magic")) == "test_name_magic"
+    assert (
+        to_snake_case(toCamelCase("numbers_33_in_the_middle_44")) == "numbers_33_in_the_middle_44"
+    )
+    assert to_snake_case(toCamelCase("eerste_h_n_id")) == "eerste_h_n_id"
+
+    assert toCamelCase(to_snake_case("numbers33inTheMiddle44")) == "numbers33InTheMiddle44"
```

Comment:
In this case, `toCamelCase` is clearly not the inverse of `to_snake_case`: `"numbers33inTheMiddle44" != "numbers33InTheMiddle44"`.

Reply:
good catch!